ubisys D1: Allow ballast configuration

felixstorm opened the pull request:
Primarily allows to control dimmer min and max levels (and could also be used for other devices if they implement the ZCL ballast configuration cluster).

Documentation will follow as well.

Thanks,  
Felix

Koenkk commented on converters/toZigbee.js:
```diff
@@ -1163,6 +1163,45 @@ const converters = {
+            for (const [attrName, attrValue] of Object.entries(value)) {
+                const attributes = {};
+                attributes[attrName] = attrValue;
+                await entity.write('lightingBallastCfg', attributes);
```
Can you also convert this to snake_case? You can copy the following function to do the conversion: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L157

Koenkk commented:
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/421

felixstorm commented:
@Koenkk Thanks for the hint to the conversion function - is it ok now like this?

I also needed to convert JSON attribute names from snake case to camel case, so I also added a function `toCamelCase`. And since I then anyway had this function, I also changed the attribute names in code to become snake case as well - just in case somebody looks at the code hoping to get the attribute names to put into the JSON...

Koenkk commented:
Thanks!
